### PR TITLE
Hotfix/referral link

### PR DESCRIPTION
**Pull request by tiffwong**
## Description

Fix referral link showing for customers who have not made a payment/met the $25 minimum threshold

## How to test

Check `/profile/referrals` for accounts that should and shouldn't be showing the referral link

### CODE REVIEW

**Review by bnussman — DISMISSED** [review]

File: packages/manager/src/features/Profile/Referrals/Referrals.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const Referrals: React.FC<CombinedProps> = (props) => {
     credit,
   } = props;
 
-  const allowReferral = url !== undefined;
+  const allowReferral = url !== '';
```
Comment (HanaXu) [inline packages/manager/src/features/Profile/Referrals/Referrals.tsx:118]: Should this be `null`? 🤔 

image

Comment (bnussman) [reply packages/manager/src/features/Profile/Referrals/Referrals.tsx:118]: Thats odd, i'm seeing this

Screen Shot 2021-07-01 at 11 44 44 AM

Comment (tiffwong) [reply packages/manager/src/features/Profile/Referrals/Referrals.tsx:118]: No, API specifically returns an empty string for the referral link so null does the same thing undefined does unfortunately

Comment (HanaXu) [reply packages/manager/src/features/Profile/Referrals/Referrals.tsx:118]: Or `const allowReferral = url ?? false;` to handle both null and undefined

Comment (DevDW) [reply packages/manager/src/features/Profile/Referrals/Referrals.tsx:118]: `const allowReferral = Boolean(url)` is probably what we'd want to do here

Comment (tiffwong) [reply packages/manager/src/features/Profile/Referrals/Referrals.tsx]: That did the trick!

**Review by HanaXu — DISMISSED** [review]

**Review by DevDW — DISMISSED** [review]

**Review by DevDW — APPROVED** [review]

**Review by HanaXu — APPROVED** [review]

**Pull request merged**